### ISSUE TITLE
Change the primary directory name?

### ISSUE THREAD

**Issue by Jen030** [ucsdCSE110wi16/CSE110W260T13#14]
Our first directory name is really long (io.github.cse110 ..... ). Can we change it? Not sure how.

**Comment by nick11roberts** [comment]
It can be changed, but the automatic refactoring in Android Studio will not perform all of the necessary changes. Aside from that, I would actually prefer to keep the directory name as it is, since it is typical to use a reverse organization URL for the package name in Android. It actually does have significance as well. This package name acts as a unique identifier in the Google Play store if we are ever to publish this. 

TL;DR 
It's a pain to change it, and it follows standard practice as it is now.
